content/en/synthetics/private_locations.md

```diff
@@ -61,6 +61,8 @@ Once you created a private location, configuring a Synthetics API test from a pr
 
     * For the Datadog US site: `curl https://api.datadoghq.com`.
     * For the Datadog EU site:   `curl https://api.datadoghq.eu`.
+    
+**Note**: Since test configurations are pulled and test results are pushed via HTTPS, you need to make sure to allow outbound traffic on port 443.
```

Comment:
```suggestion
**Note**:  You must allow inbound and outbound traffic on port `443` because test configurations are pulled and test results are pushed via HTTPS over port `443`.
```